HW4_package/source/suppl_filter_fastq.py

```diff
@@ -0,0 +1,19 @@
+
+
+def has_sufficient_gc(sequence: str, gc_bounds: tuple) -> bool:
```

Comment:
```suggestion
def is_sufficient_gc(sequence: str, gc_bounds: tuple) -> bool:
```

я бы переименовала все функции, возвращающие `bool`, в `is_`**something**, мне кажется, так логичнее.

Reply:
Здесь `is_` мне тоже нравится, но в целом `has_` тоже испольуется. 

```python
if student.has_parents():
   parents = student.get_parents()
   direcotor.invite(parents)